kitchen/bin/run.sh

```diff
@@ -6,6 +6,24 @@
 #   run.sh
 #
 # Runs Kitchen, passing through any and all arguments provided.
+# This script has special handling for an argument if it is `--mem`.
+# When the argument is `--mem`, that value is extracted and used as
+# the Xmx flag for the java command while launching kitchen.
+# The default value for memory allocated to kitchen is 128M.
+
+KITCHEN_MEM="128M"
+
+i=0
+until [ "$((i=$i+1))" -gt "$#" ]
+do
+  case "$1" in
+    --mem)
+      KITCHEN_MEM="$2"; $((i=$i-1)); shift ;;
+    *)
+      set -- "$@" "$1"  ;;
+  esac;
+  shift;
+done
```

Comment:
I needed to use `1024M`, do we want to increase this value for all tests? This will potentially cause our tests to run slower as they wait to get allocated resources.